### PR TITLE
adds env variable RAZEE_UPDATE_INTERVAL_MS for razee update interval customization

### PR DESCRIPTION
**Pull request by MajinBui**
### What has been done?
- Adds a new environment variable to modify the interval the rate razee updates
   - uses the environment variable: `RAZEE_UPDATE_INTERVAL_MS` and is optional. Defaults to `300000` (the original value at 5 mins)
   - currently set to have the lowest interval update time of `60000` (1 minute) or will error out if given a value lower.
   - Follows the same mechanisms as the other environment variables.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -71,14 +72,17 @@ function main() {
   if (!clusterId) {
     throw 'Please specify process.env.CLUSTER_ID';
   }
-  log.debug({ razeeApi, clusterId });
+  if (razeeUpdateInterval < 60000) {
+    throw `Please use a value greater than or equal to 60000. RAZEE_UPDATE_INTERVAL_MS=${RAZEE_UPDATE_INTERVAL_MS}`;
```
Comment (adamkingit) [inline src/index.js:76]: I suggest instead of throwing an error, instead log a warning and set the value to 60000 so the service still works

Comment (MajinBui) [reply src/index.js]: Updated now to default to `60000`

**Comment by MajinBui** [comment]
Seems like the Travis build is failing due to a problem with the CI process.

**Comment by MajinBui** [comment]
> while i dont think this is necessarily bad, im not sure this is a good strategic approach to your issue. Instead of relying on a quick re-apply of your CRD, you should turn reconcile=false to avoid the outage altogether.

I just tried this and I don't think this is applicable for our situation.  In our specific set up, even with `deploy.razee.io/Reconcile: "false"`, this does not work.  What happens is the **definition** for the CRD is removed which causes the specific CRD to be removed regardless of the flag.  Then, the cluster subscription will only reapply the CRD after a ceiling of 5 minutes.

```
labels:
      deploy.razee.io/Reconcile: "false"
```

**Comment by alewitt2** [comment]
so you are saying youd rather have an outage and have your CRD and all the instance of the CRD deleted instead of adding a label to all the resources?

**Comment by alewitt2** [comment]
after further consideration, i dont think this is a good idea. This interval is to catch missed websocket events, in order to ensure all the subscriptions are pointing at the correct versions, not to enforce a specific version for your application. for that, you can modify the interval for remoteresource, which is the tool that does the enforcement for each individual subscription.

**Pull request closed without merging**